packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -203,6 +203,8 @@ export class DeltaManager
     private connection: IDocumentDeltaConnection | undefined;
     private clientSequenceNumber = 0;
     private clientSequenceNumberObserved = 0;
+    // Represents the latest topmost noops sent by the client.
```

Comment:
I'd change naming / description (it was not obvious for me what it is until I looked at the rest of implementation).